app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "com.example.ikeahjalpare"
-        minSdkVersion 16
+        minSdkVersion 28
```

Comment:
I had to up the minSdkVersion to get the code to compile

Reply:
Yeah... 16 is way old.  28 is fine for this class.